force-app/main/default/classes/AFFL_AccRecordType_TEST.cls

```diff
@@ -53,10 +53,199 @@ public with sharing class AFFL_AccRecordType_TEST {
         householdRecTypeID = UTIL_Describe.getHhAccRecTypeID();
     }
 
+    private static void setupWithFieldLabels() {
+        UTIL_CustomSettings_API.getSettingsForTests(
+            new Hierarchy_Settings__c(Account_Processor__c = UTIL_Describe.getHhAccRecTypeID())
+        );
+        List<Affl_Mappings__c> mappings = [
+            SELECT ID
+            FROM Affl_Mappings__c
+            WHERE Account_Record_Type__c != NULL AND Primary_Affl_Field__c != NULL
+        ];
+        if (mappings.size() == 0) {
+            mappings.add(
```

Comment:
- [Enhancement] Abstract away affiliations mapping generation into UTIL_UnitTestData_TEST.

Reply:
Moved this code to UTIL_UnitTestData_TEST